src/main/java/org/intellij/sdk/codesync/HandleBuffer.java

```diff
@@ -55,15 +56,15 @@ public static boolean shouldSkipDiffFile(DiffFile diffFile, ConfigFile configFil
         if (!configFile.hasRepo(diffFile.repoPath)) {
             CodeSyncLogger.logEvent(String.format(
```

Comment:
This is fine. Just thinking we can do something similar to `logger.info`, `logger.debug` like `CodeSyncLogger.info`, `CodeSyncLogger.error` which will internally set the event type
I will try something like this on VSCode side

Reply:
Good suggestion, I will update the code to use have these methods instead of passing event type as arguments.

Reply:
Awesome! thanks